src/pages/zh-CN/install/auto.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 通过自动命令行来安装Astro
+description: 如何通过 create-astro CLI 工具使用 NPM、PNPM 或 Yarn 安装 Astro。
+layout: ~/layouts/MainLayout.astro
+setup: import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
+---
+准备好安装Astro了吗？跟着我们的自动或手动设置教程来开始吧。
+
+#### 前提准备
+
+- **Node.js** - `14.15.0`, `v16.0.0`,或更高版本.
+- **Text editor** - 我们建议使用带[Astro官方插件](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode)的 [VS Code](https://code.visualstudio.com/)来作为编辑器。
+- **Terminal** - Astro 可以通过其命令行界面 (CLI) 访问。
+
+<InstallGuideTabGroup />
+
+#### 安装
+
+`create-astro` 命令是从头开始一个新的 Astro 项目最快、最简单的方法。
+
+## 1. 通过命令行（CLI）运行
+
+在终端内使用 `create-astro`可以快速的启动安装向导。这将会在你运行该命令的目录下指引你一步步去创建第一个Astro项目。
+
+```shell
+# npm
+npm create astro@latest
+
+# yarn
+yarn create astro
+
+# pnpm
+pnpm create astro@latest
+```
+
+如果 `create-astro`命令成功启动, 你将会看到几个选项来作为项目起始模板：
+- `starter`: 每一个想去尝试Astro的最佳模板；
+- `minimal`: 一个包含基本功能的模板；
+- `blog, portfolio, docs, etc`: 针对特定用例而使用的主题模板。
+
+如果你选择 `starter` 参数, 你将会被要求去选择一些其他参数： [框架选择](/en/core-concepts/framework-components) (React, Svelte, Vue, Solid, Preact),如果你需要，你可以将他们添加到你的项目内。之后将会适配更多的框架。
```

Comment:
The space in line 68 needs to be removed when applying. I added it because the suggestion diff preview would break from there if it wasn't. There seems to be a problem with GitHub's rendering.

```suggestion
### 选择入门模板

你将看到可供你挑选的简短入门模板列表：
  
- `Just the basics`：对于想要探索 Astro 的人而言，这是个很好的入门模板。
- `Blog`、`Documentation`、 `Portfolio`：根据具体用例提供参考的模板。
- `Completely empty`：仅供入门使用的最小模板。

使用箭头键（向上和向下）导航到你要安装的模板，然后按回车（Enter）提交。

> 💡 想在选择前先在浏览器中查看一下模板吗？[前往 astro.new](https://astro.new/)

### 安装依赖（可选）

向导将在此时提出运行 `install` 命令的交易，你可以选择现在安装或者等会自行安装。

> ⚠️如果你此时不这样做，那么就需要在向导完成后，在开始运行项目前[安装依赖](/en/install/auto#2-install-dependencies)。

### 安装任一官方 Astro 集成（可选）

此时你可以选择添加任何[额外的 UI 框架](/en/core-concepts/framework-components)（如 React、Svelte、Vue、Solid、Preact、Lit），然后再通过运行 `astro add --yes` 添加其他 Astro 官方集成（如 Tailwind、Turbolinks、Partytown、Sitemap）。

要怎么选择想要再项目中使用的 Astro 集成？使用方向键（向上和向下）导航，空格键在选定的状态之间切换。你可以一次选择多个项目，或者你可以不选择任何集成就直接仅进行下一步。

当你对你的选择感到满意时，按回车（Enter）提交。

> 这些集成以及其他的 [Astro 社区集成](https://astro.build/integrations)，在日后也可以按照我们的[集成指南](/en/guides/integrations-guide)所写的另行添加。

在选完要添加的集成后，你应该看到类似下方的终端信息，告知你 `create-astro` 将对项目中的 `astro.config.mjs` 进行改变。

```bash
Astro will make the following changes to your config file:
	```

看到这条消息意味着你所选择的集成已经成功添加到你的项目配置中。(如果没有的话，你可以稍后手动添加它们）。

### 初始化 .git 仓库（可选）

在这最后一步，你可以选择在你的新目录下初始化 git 仓库。你可以随意选择是否要初始化，但如果你打算在项目中使用 [Git](https://git-scm.com/) 工具的话这就大有脾益。

### 下一步

当 `create-astro` 安装向导执行完毕后，你应该在屏幕上看到一些建议性说明（"Next Steps"），这些说明将帮助你完成设置并开始新项目。

```

Reply:
@kecrily If you want to fix this suggestion, you can use `~~~suggestion` at the start and `~~~` at the end instead of <code>```</code>.

Reply:
OK, I didn't even know there was such syntax for code blocks before. I only knew about indentation and ~~~.

Reply:
Yeah, I had to dig to find it when I ran into exactly the same issue you did!